### PR TITLE
47-Node.js database migration for Postgresql-Olena

### PR DESCRIPTION
**Pull request by OlenaReukova**
installed node-pg-migration for Postgresql
created migration.js for the project database

[node-pg-migrate](https://github.com/salsita/node-pg-migrate)

### CODE REVIEW

File: migrations/1688509079773_init.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        full_name VARCHAR(50),
+        cohort_id INT NOT NULL,
+        FOREIGN KEY (cohort_id) REFERENCES cohorts (id)
+    );
+  `);
+};
+
+exports.down = (pgm) => {
+	pgm.sql(`
+    DROP TABLE public.cohorts;
```
Comment (PeteLindsell) [inline migrations/1688509079773_init.js:32]: Ideally we want to delete the trainees table here too

File: migrations/1688509079773_init.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      m_3 DATE NOT NULL,
+      m_4 DATE NOT NULL,
+      m_5 DATE NOT NULL,
+      m_6 DATE NOT NULL,
+      m_7 DATE NOT NULL,
+      m_8 DATE NOT NULL
+    );
+  `);
+	pgm.sql(`
+    CREATE TABLE public.trainees (
```
Comment (PeteLindsell) [inline migrations/1688509079773_init.js:20]: Just a note that public is the default schema for a postgres database so 
`CREATE TABLE public.trainees` and `CREATE TABLE trainees` are the same

**Review by PeteLindsell — APPROVED** [review]
Looks good, adding a couple of lines about the process to the readme would be great!

File: migrations/1688509079773_init.js (hunk 2 above)
Comment (OlenaReukova) [reply migrations/1688509079773_init.js:20]: @PeteLindsell ok

File: migrations/1688509079773_init.js (hunk 1 above)
Comment (OlenaReukova) [reply migrations/1688509079773_init.js:32]: I added apportunity to delete a TABLE trainee

**Pull request merged**